### PR TITLE
DOC-1654: Update CONTRIBUTING and README

### PR DESCRIPTION
**Pull request by tiny-ben-tran**
Related Ticket: DOC-1654

Description of Changes:
* Reverted CONTRIBUTING back to md format
* Updated URL links, live demo and metadata in CONTRIBUTING
* Removed references to Jekyll
* Updated live demos section in README

Pre-checks:
- [x] Branch prefixed with `feature/6/` or `hotfix/6/`
- [x] ~`modules/ROOT/nav.adoc` has been updated (if applicable)~
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)
- [x] ~(New product features only) Release Note added~

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,619 @@
...
+
+Resources to be linked take the form of a URL following by the linked text between two brackets. Linking an external resource looks like this:
+
+```
+    hello, http://www.example.com[text to link] to the URL example.com.
+```
+
+## Live demos
+
+New live demos can be added to [modules/ROOT/examples/live-demos directory](modules/ROOT/examples/live-demos). It then can be referenced in your doc with the following code:
```
Comment (lnewson) [inline CONTRIBUTING.md:60]:
```suggestion
New live demos can be added to the [modules/ROOT/examples/live-demos directory](modules/ROOT/examples/live-demos). It then can be referenced in your doc with the following code:
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,619 @@
...
+
+```
+    liveDemo::{sub-directory-name}[]
+```
+
+
+## Adding metadata to a page
+
+Every page should have a meta section starting at line 1 containing information that is both informative and system critical. If you are creating a page or editing one that currently has inadequate meta information, please read the comments next to the default meta fields below.
```
Comment (lnewson) [inline CONTRIBUTING.md:69]:
```suggestion
Every page should have a meta section immediately after the heading containing information that is both informative and system critical. If you are creating a page or editing one that currently has inadequate meta information, please read the comments next to the default meta fields below.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,619 @@
...
+
+```
+    :navtitle:          // Descriptive title for the navigation bar, if omitted nav defaults to title field.
+    :description_short: // Short description for the grid layouts.
```
Comment (lnewson) [inline CONTRIBUTING.md:73]:
```suggestion
```
This isn't a valid property in AsciiDoc and was a custom one used by the old system.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,619 @@
...
+    :description:       // Detailed description at page level for the header section under the title.
+    :keywords:          // Space-separated list of keywords in the content.
```
Comment (lnewson) [inline CONTRIBUTING.md:75]:
```suggestion
    :keywords:          // Comma-separated list of keywords in the content.
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,619 @@
...
+
+We welcome people with technical writing experience to help make TinyMCE’s docs as useful to our community as possible. Although you don’t need to be an expert developer to contribute, a basic understanding of software development is helpful.
+
+What is more important is your ability to articulate complex ideas. Our goal is to help developers understand TinyMCE’s functionality as quickly as possible.
+
+Simply branch the [docs GitHub repository](https://github.com/tinymce/tinymce-docs) and submit a PR, or reach out to the [TinyMCE docs team](https://github.com/tinymce/tinymce-docs/issues/new?assignees=&labels=question&template=question.md) if you have a specific question.
+
+# TinyMCE documentation tech stack
+
+The documentation is built with [Antora](https://antora.org/) and text files are written in [ascii doc](https://asciidoc.org/). You will find the [repo here](https://github.com/tinymce/tinymce-docs).
```
Comment (lnewson) [inline CONTRIBUTING.md:17]:
```suggestion
The documentation is built with [Antora](https://antora.org/) and text files are written in [AsciiDoc](https://asciidoc.org/). You will find the [repo here](https://github.com/tinymce/tinymce-docs).
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,619 @@
...
+
+## Highlighting
+
+Code language syntax and associated elements are highlighted using an open and closing backtick. For example, this `object` is actually enclosed with backticks like this: `object`.
+
+Highlight words relating specifically to executable parts of software languages. Keys and their values when pertaining to objects and methods (particularly the case with TinyMCE) should have backticks. We cannot cover every use case here and contributors need to use their best judgment. What is more important is your ability to articulate complex ideas.
+
+## Code snippets
+
+Code snippets should always be complete, executable code blocks starting with the `tinymce` object. We write code snippets by enclosing the example code within three opening and closing backticks. The first set of backticks is immediately followed by the code’s language, which helps correct syntax highlighting: typically, html, css, js (and sometimes php).
```
Comment (lnewson) [inline CONTRIBUTING.md:27]: This section is incorrect... this isn't how code snippets are done in AsciiDoc. It uses something like this instead:
```
[source, js]
----
    tinymce.init({
      selector: 'textarea', // change this value according to your HTML
      plugins: 'codesample',
      toolbar: 'codesample',
    });
----
```

- [ ] Fix this section

File: README.md [hunk 7]
```diff
@@ -85,11 +85,11 @@ Live demos can be added to content using the "live-demo.html" include, such as:
 
 There are several options for this include file - please see live-demo.html for details.
 
-The source for each live demo is store in the `_includes/live-demos/` directory.
+The source for each live demo is store in the `modules/ROOT/examples/live-demos/` directory.
 
 #### Overriding the tinymce URL in live demos
 
-All live demos usually get their tinymce.min.js URL from the `tinymce_live_demo_url` setting in the _config.yml file.
+All live demos usually get their `tinymce.min.js` URL from the `tinymce_live_demo_url` setting in the `antora.yml` file.
```
Comment (lnewson) [inline README.md:92]: The section above this needs fixing as it still uses the old demo syntax and mentions it's a WIP.

- [ ] Fix live demo section above

File: README.md [hunk 8]
```diff
@@ -98,7 +98,7 @@ However, there are some instances where you wish to override this, e.g.
 To help with this, there are two mechanisms for overriding the tinymce.min.js URL.
 
  1. Set `tinymce_live_demo_url` in `_config-local-override.yml`.
-    - This will override the `tinymce_live_demo_url` setting in `config.yml`.
+    - This will override the `tinymce_live_demo_url` setting in `antora.yml`.
     - This file is *not* intended to be checked in.
     - This option changes the URL for all live demos.
```
Comment (lnewson) [inline README.md:104]: The part below needs a minor update, as `live-demo.html` doesn't exist anymore. It should probably just say something like `when including the live demo.` instead I guess 🤷

File: README.md [hunk 9]
```diff
@@ -98,7 +98,7 @@ However, there are some instances where you wish to override this, e.g.
 To help with this, there are two mechanisms for overriding the tinymce.min.js URL.
 
  1. Set `tinymce_live_demo_url` in `_config-local-override.yml`.
```
Comment (lnewson) [inline README.md:100]: This doesn't work anymore... if you want to override it, you need to add something like this to the antora dev playbook instead:
```
  attributes:
    tinymce_live_demo_url: https://cdn.tiny.cloud/1/qagffr3pkuv17a8on1afax661irst1hbr4e6tbv888sz91jc/tinymce/6-dev/tinymce.min.js
```

File: CONTRIBUTING.md [hunk 10]
```diff
@@ -0,0 +1,624 @@
...
+
+## Highlighting
+
+Code language syntax and associated elements are highlighted using an open and closing backtick. For example, this `object` is actually enclosed with backticks like this: `object`.
+
+Highlight words relating specifically to executable parts of software languages. Keys and their values when pertaining to objects and methods (particularly the case with TinyMCE) should have backticks. We cannot cover every use case here and contributors need to use their best judgment. What is more important is your ability to articulate complex ideas.
+
+## Code snippets
+
+Code snippets should always be complete, executable code blocks starting with the `tinymce` object. We write code snippets using [source blocks](https://docs.asciidoctor.org/asciidoc/latest/verbatim/source-blocks/) which starts by specifying the `source` style and the code snippet language in square brackets to enable syntax highlighting. The language is typically html, css, js, or ts.
```
Comment (tiny-ben-tran) [inline CONTRIBUTING.md:27]: @metricjs Can you proofread this paragraph?

**Review by lnewson — APPROVED** [review]

File: CONTRIBUTING.md [hunk 11]
```diff
@@ -0,0 +1,619 @@
...
+
+- `<html>`
+
+- `var React = require('react');`
+
+When referring to a program, use the proper name.
+
+For example:
+
+- "Microsoft Internet Explorer" not "IE"
```
Comment (brian-forte) [inline CONTRIBUTING.md:174]:
```suggestion
- “Google Chrome” not “Chrome”
```

Given Internet Explorer has been deprecated for use in TinyMCE 6 (and is, generally and finally, no longer being widely used), a more current example programme’s proper name seems apropos.

File: CONTRIBUTING.md [hunk 12]
```diff
@@ -0,0 +1,619 @@
...
+
+2.  IMPORTANT: Use when ignoring the notice *may* or *will* lead to unexpected behavior.
+
+3.  CAUTION: Use when ignoring the notice *may* lead to one or more of the following:
+
+    - A significant increase in the risk of a security breach.
+
+    - Create a security vulnerability.
+
+    - Cause information loss.
```
Comment (brian-forte) [inline CONTRIBUTING.md:333]:
```suggestion
    - Information loss.
```

For each item in the list that follows _one or more of the following_ to, in fact, **be** _one or more of the following_, said items must be things: nouns or noun phrases. _Cause information loss_ is an action; a verb phrase.

File: CONTRIBUTING.md [hunk 13]
```diff
@@ -0,0 +1,619 @@
...
+
+    - System failure.
+
+    - Worse outcomes than those listed here.
+
+4.  WARNING: Use when ignoring the notice *will* lead to one or more of the following:
+
+    - A significant increase in the risk of a security breach.
+
+    - Create a security vulnerability.
```
Comment (brian-forte) [inline CONTRIBUTING.md:343]:
```suggestion
    - A security vulnerability.
```

For each item in the list that follows _one or more of the following_ to, in fact, **be** _one or more of the following_, said items must be things: nouns or noun phrases. _Create a security vulnerability_ is an action; a verb phrase.

File: CONTRIBUTING.md [hunk 14]
```diff
@@ -0,0 +1,619 @@
...
+
+    - Cause information loss.
```
Comment (brian-forte) [inline CONTRIBUTING.md:345]:
```suggestion
    - Information loss.
```

For each item in the list that follows one or more of the following to, in fact, be one or more of the following, said items must be things: nouns or noun phrases. Cause information loss is an action; a verb phrase.

File: CONTRIBUTING.md [hunk 15]
```diff
@@ -0,0 +1,619 @@
...
+
+- For a brief overview of how users read (or don’t read) content, see: [Content Design London: Readability Guidelines - How people read](https://readabilityguidelines.co.uk/content-design/how-people-read/).
+
+- For points on how to structure content, see: [Content Design London: Readability Guidelines - Page design](https://readabilityguidelines.co.uk/content-design/page-design/).
+
+- For general information on writing, covering a variety of subjects, see: [plainlanguage.gov: Mary Dash’s Writing Tips](https://plainlanguage.gov/resources/articles/dash-writing-tips/).
+
+# Glossary
+
+Accessible Rich Internet Applications (ARIA) Acronym for *Accessible Rich Internet Applications* Spell out and describe on first mention within a section.
```
Comment (brian-forte) [inline CONTRIBUTING.md:364]:
```suggestion
Accessible Rich Internet Applications (ARIA): Acronym for *Accessible Rich Internet Applications*. Spell out and describe on first mention within a section.
```

Missing punctuation.

File: CONTRIBUTING.md [hunk 16]
```diff
@@ -0,0 +1,619 @@
...
+add-on
+: Do not use *add-on*. Use *"`plugin`"* instead.
+
+anchor
+: Use when describing the Anchor plugin or the technical insertion of an
+anchor tag into HTML. When talking with a business audience, use
+*bookmark* or add *bookmark* as a clarifier.
+
+Application Programming Interface (API)  
+: Use *API* in most circumstances. Spell it out on first mention only if kthe document is for a non-technical audience.
```
Comment (brian-forte) [inline CONTRIBUTING.md:377]:
```suggestion
: Use *API* in most circumstances. Spell it out on first mention only if the document is for a non-technical audience.
```

Typo correction.

File: CONTRIBUTING.md [hunk 17]
```diff
@@ -0,0 +1,619 @@
...
+Tiny 4.x, Tiny 5.0  
+: *Tiny* is not a product. Do not give it a version. Use the product name, such as TinyMCE 5.0 or TinyMCE 4.9.
+
+Tiny Cloud  
+: Correct way to refer to the Tiny Cloud service. Spell it out on all occasions.
+
+For example: *Tiny Cloud* includes a fast content delivery network.
+
+Tiny Technologies Inc.  
+: *Tiny Technologies Inc.* is full and correct business name for *Tiny*
```
Comment (brian-forte) [inline CONTRIBUTING.md:559]:
```suggestion
: *Tiny Technologies Inc.* is the full and correct business name for *Tiny*
```

Missing definite article.

File: CONTRIBUTING.md [hunk 18]
```diff
@@ -0,0 +1,619 @@
...
+
+Do not use *Ephox Corporation*.
+
+TinyMCE  
+: This is the JavaScript library for rich text editing.
+
+Capitalize the *MCE* Do not use *Tinymce* *tinymce* is allowed in code
```
Comment (brian-forte) [inline CONTRIBUTING.md:566]:
```suggestion
Capitalize the *MCE* Do not use *Tinymce*. *tinymce* is allowed in code
```

Missing full-stop.

File: CONTRIBUTING.md [hunk 19]
```diff
@@ -0,0 +1,619 @@
...
+and codeblocks where required.
+
+TinyMCE 4
```
Comment (brian-forte) [inline CONTRIBUTING.md:569]:
```suggestion
TinyMCE 5  
```

Update the glossary’s version number examples to reflect the release of *TinyMCE 6*.

File: CONTRIBUTING.md [hunk 20]
```diff
@@ -0,0 +1,619 @@
...
+Do not use *Ephox Corporation*.
+
+TinyMCE  
+: This is the JavaScript library for rich text editing.
+
+Capitalize the *MCE* Do not use *Tinymce* *tinymce* is allowed in code
+and codeblocks where required.
+
+TinyMCE 4  
+: Correct way to refer to the 4.x releases of TinyMCE unless a specific version number (major.minor.patch) is *required*.
```
Comment (brian-forte) [inline CONTRIBUTING.md:570]:
```suggestion
: Correct way to refer to the 5.x releases of TinyMCE unless a specific version number (major.minor.patch) is *required*.
```

Update the glossary’s version number examples to reflect the release of *TinyMCE 6*.

File: CONTRIBUTING.md [hunk 21]
```diff
@@ -0,0 +1,619 @@
...
+
+TinyMCE 4.x
```
Comment (brian-forte) [inline CONTRIBUTING.md:572]:
```suggestion
TinyMCE 5.x  
```

Update the glossary’s version number examples to reflect the release of *TinyMCE 6*.

File: CONTRIBUTING.md [hunk 22]
```diff
@@ -0,0 +1,619 @@
...
+: This is the JavaScript library for rich text editing.
+
+Capitalize the *MCE* Do not use *Tinymce* *tinymce* is allowed in code
+and codeblocks where required.
+
+TinyMCE 4  
+: Correct way to refer to the 4.x releases of TinyMCE unless a specific version number (major.minor.patch) is *required*.
+
+TinyMCE 4.x  
+: Do not use. Use *TinyMCE 4* or *version 4* If necessary to specify the availability of a new feature or change in behavior, specify the minor version number such as TinyMCE 4.7.
```
Comment (brian-forte) [inline CONTRIBUTING.md:573]:
```suggestion
: Do not use. Use *TinyMCE 5* or *version 5* If necessary to specify the availability of a new feature or change in behavior, specify the minor version number such as TinyMCE 5.7.
```

Update the glossary’s version number examples to reflect the release of *TinyMCE 6*.

File: CONTRIBUTING.md [hunk 23]
```diff
@@ -0,0 +1,619 @@
...
+
+TinyMCE 5
```
Comment (brian-forte) [inline CONTRIBUTING.md:575]:
```suggestion
TinyMCE 6  
```

Update the glossary’s version number examples to reflect the release of *TinyMCE 6*.

File: CONTRIBUTING.md [hunk 24]
```diff
@@ -0,0 +1,619 @@
...
+and codeblocks where required.
+
+TinyMCE 4  
+: Correct way to refer to the 4.x releases of TinyMCE unless a specific version number (major.minor.patch) is *required*.
+
+TinyMCE 4.x  
+: Do not use. Use *TinyMCE 4* or *version 4* If necessary to specify the availability of a new feature or change in behavior, specify the minor version number such as TinyMCE 4.7.
+
+TinyMCE 5  
+: Correct way to refer to the 5.x releases of TinyMCE unless a specific version number (major.minor.patch) is *required*.
```
Comment (brian-forte) [inline CONTRIBUTING.md:576]:
```suggestion
: Correct way to refer to the 6.x releases of TinyMCE unless a specific version number (major.minor.patch) is *required*.
```

Update the glossary’s version number examples to reflect the release of *TinyMCE 6*.

File: CONTRIBUTING.md [hunk 25]
```diff
@@ -0,0 +1,619 @@
...
+Do not use *UI experience* It is either *user experience (UX)* OR *user interface (UI)*, but never both.
+
+web  
+: Do not use *web* Use *Internet* for clarity. Can be used for terms such as *web server*.
+
+Web Content Accessibility Guidelines (WCAG)  
+: Acronym for *Web Content Accessibility Guidelines* Spell out and describe on first mention within a section.
+
+WYSIWYG  
+: Avoid using *WYSIWYG* Use *rich text editor*
```
Comment (brian-forte) [inline CONTRIBUTING.md:622]:
```suggestion
: Avoid using *WYSIWYG*. Use *rich text editor*.
```

Missing puncutation.

File: CONTRIBUTING.md (hunk 10 above)
Comment (brian-forte) [inline CONTRIBUTING.md:27]:
```suggestion
Code snippets should be complete, executable code blocks starting with the `tinymce` object. We write code snippets using [source blocks](https://docs.asciidoctor.org/asciidoc/latest/verbatim/source-blocks/). These start by specifying the `source` style and the code snippet language in square brackets to enable syntax highlighting. The language is typically html, css, js, or ts.
```

A @brian-forte proof reading copy-edit in lieu of @metricjs. ‘Always’ removed as unnecessary in the first sentence. And the second sentence contained a run-on clause that is now a separate sentence.

File: CONTRIBUTING.md [hunk 26]
```diff
@@ -0,0 +1,620 @@
...
+
+- For a brief overview of how users read (or don’t read) content, see: [Content Design London: Readability Guidelines - How people read](https://readabilityguidelines.co.uk/content-design/how-people-read/).
+
+- For points on how to structure content, see: [Content Design London: Readability Guidelines - Page design](https://readabilityguidelines.co.uk/content-design/page-design/).
+
+- For general information on writing, covering a variety of subjects, see: [plainlanguage.gov: Mary Dash’s Writing Tips](https://plainlanguage.gov/resources/articles/dash-writing-tips/).
+
+# Glossary
+
+Accessible Rich Internet Applications (ARIA): Acronym for *Accessible Rich Internet Applications*. Spell out and describe on first mention within a section.
```
Comment (lnewson) [inline CONTRIBUTING.md:362]:
```suggestion
Accessible Rich Internet Applications (ARIA)
: Acronym for *Accessible Rich Internet Applications*. Spell out and describe on first mention within a section.
```
This section is meant to be a description list, so this should be on a new line.

**Review by metricjs — APPROVED** [review]

File: CONTRIBUTING.md [hunk 27]
```diff
@@ -0,0 +1,614 @@
...
+
+# TinyMCE documentation tech stack
+
+The documentation is built with [Antora](https://antora.org/) and text files are written in [AsciiDoc](https://asciidoc.org/). You will find the [repo here](https://github.com/tinymce/tinymce-docs).
+
+## Highlighting
+
+Code language syntax and associated elements are highlighted using an open and closing backtick. For example, this `object` is actually enclosed with backticks like this: `object`.
+
+Highlight words relating specifically to executable parts of software languages. Keys and their values when pertaining to objects and methods (particularly the case with TinyMCE) should have backticks. We cannot cover every use case here and contributors need to use their best judgment. What is more important is your ability to articulate complex ideas.
```
[the remark this answers is not in the document]
Comment (tiny-ben-tran) [reply CONTRIBUTING.md:23]: Agreed. I deleted it.

**Pull request merged**